README.md

```diff
@@ -4,47 +4,30 @@ This monorepo comprises a suite of tools **currently in development** to allow d
 
```

Comment:
I for my part stumble upon the Ultralight header a bit always.

<img width="799" alt="grafik" src="https://github.com/ethereumjs/ultralight/assets/931137/26799ee0-ffda-4232-94a8-83a113f97d51">

Particulary upon the "Clients" in plural and the "in the browser" parts.

The whole things just reads a bit strange.

Maybe "TypeScript" is the more important/graspable/(and: "real" for the moment) part here atm? 🤔

And "Client" instead of "Clients".

So I would suggest: "Portal Network Client in TypeScript"

But you have likely also some thoughts about that.

Reply:
Hehe, definitely, given that we've sort of deprecated the browser client for now anyway.  Will do some further revisions based on the above feedback.  Thanks!